### ISSUE TITLE
Explicit is better than implicit

### ISSUE THREAD

**Issue by integrii** [go-proverbs/go-proverbs.github.io#30]

**Comment by petar-dambovaliev** [comment]
I like it.

**Comment by nnutter** [comment]
Couple of problems:

- Interfaces are implicitly satisfied.
- This is a mantra (albeit almost tounge-in-cheek) of the [Zen of Python](https://www.python.org/dev/peps/pep-0020/).

**Comment by robpike** [comment]
To add a proverb, one needs context that explains it. The originals came from a talk I gave. ("Don't panic" was added by me immediately after, but should have been put on hold.) The hope is that any further additions would be not just gnomic sayings (I'm not even sure what this one means in general, as @nnutter  has hinted) but distillations of arguments about how to use the language, and those arguments need to be presented.

In other words, until there is another proverbs talk or maybe book, explaining and expanding on  the originals plus the additions, I'd prefer to see no additions to the list.

**Comment by nnutter** [comment]
@robpike Would it be useful to you to express that in a [`CONTRIBUTING.md`](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) so people would be linked to that when trying to create a PR or an issue?

**Comment by robpike** [comment]
Happy to see that added.
